src/store/staking/staking.test.ts

```diff
@@ -468,4 +472,124 @@ describe('staking store', () => {
       expect(runResult.effects).toEqual({});
     });
   });
+
+  describe('fetchHistoryStakingForContract', () => {
+    const txHashes = ['0x05644455', '0x3443'];
+
+    const combinedHistoryStakesList: StakingHistoryListItem[] = [
+      {
+        asset: 'FISH',
+        unlockDate: dates[0].toString(),
+        stakedAmount: stakes[0],
+        totalStaked: stakes[0],
+        txHash: txHashes[0],
+      },
+      {
+        asset: 'FISH',
+        unlockDate: dates[1].toString(),
+        stakedAmount: stakes[0],
+        totalStaked: stakes[1],
+        txHash: txHashes[1],
+      },
+    ];
+
+    const successState: DeepPartial<RootState> = {
+      ...initialState,
+      [Reducers.Staking]: {
+        ...initialState[Reducers.Staking],
+        stakesListHistory: {
+          state: 'success',
+          data: combinedHistoryStakesList,
+        },
+      },
+    };
+
+    const failureState: DeepPartial<RootState> = {
+      ...initialState,
+      [Reducers.Staking]: {
+        ...initialState[Reducers.Staking],
+        stakesListHistory: { state: 'failure', data: [] },
+      },
+    };
+
+    const vestAddress = '0x94e907A6483b5ef';
+
+    const addresses = [vestAddress, testAccount];
+
+    const stakeEvents = [
```

Comment:
now that we're filtering the list on the query level there is no need to have both `stakeEvents` and `filteredStakeEvents` in tests. We can test if query is triggered with proper arguments and assume it's returning proper, filtered data

Reply:
Actually you're already testing it the way I described, but you could have just one variable with stakeEvents